src/components/runtime-common.js

```diff
@@ -124,7 +124,10 @@ export const SaveFilesHelp = () => {
 }
 
 export const GalaxyWarning = () => {
-  return p([div({ style: { fontWeight: 600 } }, ['Important: Please keep this tab open and logged in to Terra while using Galaxy.']), ' Galaxy will open in a new tab. '])
+  return div({ style: { fontWeight: 600 } },
+    [p('Important: Please keep this tab open and logged in to Terra while using Galaxy.'),
+      p('Galaxy will open in a new tab. ')]
+  )
```

Comment:
What I meant here was that this was valid before, although a bit much to have on one line. You should be able to revert these lines, and maybe just add some line breaks:
```suggestion
  return p([
    div({ style: { fontWeight: 600 } }, ['Important: Please keep this tab open and logged in to Terra while using Galaxy.']),
    'Galaxy will open in a new tab.'
  ])
```

Reply:
react complains when you nest tags in a `<p>`, I'd prefer to clean up the console

Reply:
Ah, my bad, I didn't realize React was complaining about it. In that case, I'd still like to clean up the spacing here:
```suggestion
  return h(Fragment, [
    p({ style: { fontWeight: 600 } }, 'Important: Please keep this tab open and logged in to Terra while using Galaxy.'),
    p('Galaxy will open in a new tab.')
  ])
```

Just note that this will still change the spacing between the text from what it was before this PR. I've moved the style so that only the section that's currently bold will be bold.